consensus-types/forks/eip4844/BUILD.bazel

```diff
@@ -1,14 +0,0 @@
-load("@prysm//tools/go:def.bzl", "go_library")
-
-go_library(
-    name = "go_default_library",
-    srcs = ["tx.go"],
-    importpath = "github.com/prysmaticlabs/prysm/v3/consensus-types/forks/eip4844",
-    visibility = ["//visibility:public"],
-    deps = [
-        "@com_github_ethereum_go_ethereum//common:go_default_library",
-        "@com_github_ethereum_go_ethereum//core/types:go_default_library",
-        "@com_github_protolambda_ztyp//codec:go_default_library",
-        "@com_github_pkg_errors//:go_default_library",
-    ],
-)
```

Comment:
deleted intentionally?

Reply:
yeah that directory is now empty, forgot to blow this away in my last PR.